app/presenters/chf/file_set_presenter.rb

```diff
@@ -20,6 +20,12 @@ def representative_file_id
       end
     end
 
+    def representative_checksum
+      # I can not explain why this didn't work with a delegate to solr_document
```

Comment:
When you delegate to solr_document you still have to write an accessor method in the solr_document model. To bridge to the solr_name.